tests/test_codemoduleapi.py

```diff
@@ -25,16 +25,26 @@ def test_instance_method_does_not_convert(self, standalone_tsm_context):
 
     @classmethod
     @code_module
-    def test_class_method_converts(cls, standalone_tsm_context_com_object):
+    def test_class_method_converts_core(cls, standalone_tsm_context_com_object):
```

Comment:
Because this test method is not called automatically by pytest, would it make sense to not add the test_ prefix to it?

Reply:
That seems fair to me to be explicit.

Reply:
https://github.com/ni/nitsm-python/pull/140/commits/a7e7a2bf20b1f0eb26df0fdf70bcfb46ae9f4151